### DISCUSSION TITLE
Authentication for token introspection endpoint

### DISCUSSION THREAD

**Discussion by TimoKramer** [kanidm/kanidm#2664] [Q&A] [answered]
Hi,
do you have some documentation how I can authenticate to the token introspection endpoint? I tried with the header `"authorization" "Bearer <access_token>"` where access_token is the token received by a successful call to the token url like explained in [RFC 7662](https://datatracker.ietf.org/doc/html/rfc7662#section-2.1). How do I do that?

**Comment by TimoKramer** [comment, marked as answer]
Ok, basic-auth with client-id and client-secret works. Sorry for bothering.

**Reply by Firstyear** [reply]
No problem :) glad you solved it.
